config/compiler.gfortran.mk

```diff
@@ -26,7 +26,7 @@ FMAKEDEP = $(SCRIPTS_DIR)/sfmakedepend
 CPPFLAGS += -DCOMPILER_G95
 FFLAGS = -g -cpp -fconvert=big-endian -O2 -fno-range-check -fallow-argument-mismatch
 F90FLAGS = -g -cpp -fconvert=big-endian -O2 -fno-range-check -ffree-line-length-none -fallow-argument-mismatch
-LFLAGS =
+LFLAGS = -nostartfiles -Wno-main
```

Comment:
These (and others) need to be wrapped in `ifdef`s for only when we are compiling with GC.